client/src/Asset.tsx

```diff
@@ -173,29 +217,29 @@ function Asset(): JSX.Element {
                     </p>
                 </div>
                 <div className={classes.rightPanel}>
-                    <Typography variant="h3">
+                    <Typography className={classes.heading} variant="h3">
                         {asset.title}
                     </Typography>
                     <Typography className={classes.subText} variant="subtitle1">
                         {asset.categories.join(', ')}
                     </Typography>
-                    <Typography className={classes.subText} variant="subtitle1">
-                        Posted By {asset.postedBy}
-                    </Typography>
+                    {postedFrom()}
                     <Typography className={classes.subText} variant="subtitle1">
                         <RoomOutlined />{asset.location}
                     </Typography>
                     <Typography className={classes.subText} variant="subtitle1">
                         <TodayOutlined />{asset.datePosted}
                     </Typography>
+                    {aboutInfo()}
```

Comment:
i would've said just change to the snippet below. but i think i see why you're trying to use these subrenders, because this render is getting lengthy?
if that's the case then i would say we break this into a data-wrapper and display-component pattern. which is this: we have this `Asset` component that handles all of the state and logic etc. and the only styling it handles is the spacing between the 2 children, which would be those display-components aka something like `AssetViewLeftPanel` and `AssetViewRightPanel`, and these kiddos are really dumb components, they shouldn't have anything but a render/return in the function if possible
so yeah this seems like a bigger change so maybe do the below snippet for this PR and then we can clean it up in a fast follow PR? lmk what you think
```
{asset.organization && (
                  <div>
                    <Typography className={classes.linkedText} variant="subtitle1">
                    About the Nonprofit
                    </Typography>
                    <Typography className={classes.subText} variant="subtitle1">
                    {asset.description}
                    </Typography>
                </div>
)}
```